modules/test/playwright/pages/layout-admin-web/PagesAdminPage.ts

```diff
@@ -233,28 +235,90 @@ export class PagesAdminPage {
 		await this.saveConfiguration();
 	}
 
+	async expectThemeToBeDeactivated(themeName: string) {
+		const themeCard = this.getThemeCard(themeName);
+
+		await reloadUntilNotVisible({
+			beforeReload: async () => await this.openThemeSelector(),
+			maxAttempts: 10,
+			myLocator: themeCard,
+			page: this.page,
+		});
+
+		await expect(themeCard).toBeHidden();
+	}
+
+	async expectThemeToBeActivated(themeName: string) {
```

Comment:
Since these expect functions are only used by our theme tests it makes more sense to keep them in those tests.